src/controllers/UserController.ts

```diff
@@ -117,6 +119,41 @@ export class UserController {
 
     return { role: queriedRoleFromId };
   }
+
+  @Get('/:userID/inductee-points')
+  @UseBefore(InducteeAuthMiddleware)
+  @ResponseSchema(AppUserInducteePointsResponse)
+  @OpenAPI({ security: [{ TokenAuth: [] }] })
+  async getUserInducteePoints(
+    @Param('userID') userID: number,
+    @CurrentUser({ required: true }) requestAppUser: AppUser
+  ): Promise<AppUserInducteePointsResponse | undefined> {
+    if (this.appUserService.isInvalidNonOfficerAccess(requestAppUser, userID)) {
+      throw new ForbiddenError();
+    }
+
+    const points = await this.appUserService.getInducteePoints(userID);
+
+    // I don't like this but I'll take it for now - Godwin Oct 24 2020
+    return points;
```

Comment:
Perhaps renaming it to pointsObj or smth is better? There is a bit more information that just the points themselves.

Reply:
idr why i did this don't ask :|

Reply:
I think points is fine for now... it's mainly just points right? idt pointsObj is strictly a better name, maybe induction requirements might be better lol

Reply:
Yeah sounds good, I'll approve this then